client/src/frontend/components/project/Detail.js

```diff
@@ -33,7 +33,7 @@ class Detail extends Component {
     return (
       <>
         <section>
-          <Hero project={project} />
+          <EntityHero entity={project} />
```

Comment:
It wasn't working the client for me. I think this needs to be updated to show the Project pattern.